tools/devops/automation/scripts/TestResults.psm1

```diff
@@ -95,7 +95,7 @@ class TestResult {
     }
 
     [string] GetLabelSuffix() {
-        if ($this.Title.EndsWith("_Multiple")) {
+        if ($this.Title.ToLowerInvariant().EndsWith("_multiple")) {
```

Comment:
NBD for this code, but I believe passing StringComparison is generally faster (no allocations needed):

```suggestion
        if ($this.Title.EndsWith("_multiple", StringComparison.OrdinalIgnoreCase)) {
```

Reply:
Unfortunately, the `EndsWith` in PowerShell does not take a `StringComparision` argument. I can switch to using `-like` instead since that is case insensitive by default.

Reply:
Ah, pwsh quacks like C#, but isn't quite the same... 😒